app/models/bot/keep.rb

```diff
@@ -17,7 +17,8 @@ def self.archiver_annotation_types
       'keep_backup',    # VideoVault
```

Comment:
Is `keep_backup` still active on either Pender or Check API ?

Reply:
It was removed from Pender. On CheckAPI it is not active now, there might be only the old annotations

Reply:
OK thanks will file the a refactoring to review and remove Video Vault archives.